src/ClassifierHelpers.ts

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable security/detect-object-injection */
+// Disabling this since the standard object we will pass will have no prototype chain
+// thus, it will return undefined for properties not present in the object itself
+
+/**
+ * Changes the last element of array to `*`
+ * @param path The path in form of an array
+ * @returns The path in form of a string with `*` at the end
+ *
+ * @example
+ * changeLastElementToPlaceholder(['', 'servers', 'production']); // '/servers/*'
+ */
+export function changeLastElementToPlaceholder(path: string[]): string {
+  path.pop();
+  path.push('*');
+  return path.join('/');
+}
+
+/**
+ * Generates the correct path to get the classification data from standard
+ * @param standard The standard object
+ * @param path The path to find
+ * @returns The path found
+ */
+export function generateClassifierPath(
+  standard: any,
```

Comment:
why standard is any? shouldn't we have an interface for it? or will it follow once standard-related PR is merged?

Reply:
Correct. I'm gonna make related to the standard, and then we will define an interface for it.